site2/docs/client-libraries-cpp.md

```diff
@@ -14,7 +14,15 @@ Pulsar C++ client is supported on **Linux** ,**MacOS** and **Windows** platforms
 
 [Doxygen](http://www.doxygen.nl/)-generated API docs for the C++ client are available [here](/api/cpp).
 
-## System requirements
+
+## Linux
+
+> **Note**   
+> You can choose one of the following three installation methods according to your own situation: Compilation, Install RPM or Install Debian.
```

Comment:
```suggestion
> You can choose one of the following installation methods based on your needs: Compilation, Install RPM or Install Debian.
```

Reply:
Hi @zhaoyajun2009 you've ignored my comments, any thoughts？

Reply:
Solved, please review